src/rest/src/main/java/org/geoserver/rest/RestControllerAdvice.java

```diff
@@ -98,6 +98,7 @@ public void handleRestException(
         notifyExceptionToCallbacks(request, response, e);
 
         if (e.getStatus().is4xxClientError()) {
+            /* TODO: this prevents errors responses to PUT requests to be returned to the caller */
```

Comment:
Avoid adding random TODOs. Since it was added, please explain why it's preventing error responses for PUT requests.